src/main/java/uk/gov/hmcts/reform/roleassignment/util/ValidationUtil.java

```diff
@@ -180,4 +185,44 @@ public static void validateCaseId(String caseId) {
     public static boolean csvContains(String value, String csv) {
         return Arrays.stream(csv.split(",")).map(String::trim).anyMatch(value::equals);
     }
+
+    public static MultipleQueryRequest validateQueryRequests(List<QueryRequest> queryRequests) {
```

Comment:
Are we using this returned MultipleQueryRequest any where, if not then make it void.